frontend/coprs_frontend/tests/test_exceptions.py

```diff
@@ -11,6 +11,7 @@
 class TestExceptionHandling(CoprsTestCase):
     def test_json_only_for_api(self):
         app.config["SESSION_COOKIE_DOMAIN"] = "localhost.localdomain"
+        app.config["SESSION_TYPE"] = "redis"
```

Comment:
Can you document why this is needed?

Reply:
This is leftover from debugging, not needed.